ggshield/sca/output/schemas.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass, field
+from typing import List, Optional, Type, cast
+
+import marshmallow_dataclass
+from pygitguardian.models import Base, BaseSchema, FromDictMixin
+
+
+@dataclass
+class SCAVulnerability(Base, FromDictMixin):
+    severity: str
+    summary: str
+    cve_ids: List[str] = field(default_factory=list)
+    created_at: Optional[str] = None
```

Comment:
```suggestion
    created_at: datetime | None
```

I think we can do this with ggshield's python version. Also this is a datetime object in backend

Reply:
Jumping in on that topic: for the typing, we must stick to the "old" way to ensure that ggshield is compatible with previous versions of Python (3.7+), so you cannot use: `|`, "dict", "list" etc...

What I don't understand is why `created_at` is a `str` instead of a `datetime` ? Is is cast later ?